packages/@sanity/components/src/buttons/DropDownButton.js

```diff
@@ -175,14 +144,13 @@ class DropDownButton extends React.PureComponent {
                         ref={this.setMenuElement}
                         style={{minWidth: `${width}px`}}
                       >
-                        <Escapable onEscape={event => ((isActive || event.shiftKey) && this.handleCloseMenu())} />
+                        <Escapable onEscape={event => ((isActive || event.shiftKey) && this.handleClose())} />
                         <Menu
                           items={items}
                           isOpen
                           className={menuClassName}
                           onAction={this.handleAction}
-                          onClickOutside={this.handleCloseMenu}
-                          onClose={this.handleCloseMenu}
+                          onClickOutside={event => ((isActive || event.shiftKey) && this.handleClose())}
```

Comment:
Does this make sense? Should we support `Shift + Click outside` to close all? If so we should support it all over. If not this line should instead be
`onClickOutside={isActive && this.handleClose}`

Reply:
Agreed.

Reply:
Agreed on what? :)